firstvoices/backend/tasks/alphabet_tasks.py

```diff
@@ -107,9 +107,7 @@ def recalculate_custom_order(site_slug: str):
         new_order = alphabet.get_custom_order(cleaned_title)
 
         # Save the entry to recalculate custom order and clean title
-        entry.title = cleaned_title
-        entry.custom_order = new_order
-        entry.save()
+        entry.save(set_modified_date=False)
```

Comment:
I'm not sure if this got missed, but do we want to remove the updation of title and custom_order here ?

Reply:
These functions are automatically called in `DictionaryEntry.save()`! The reason why they are in the code is presumably because Bronson had refactored this to use `update` instead and then it got changed back to `save`?

Reply:
Oh, got it. That makes sense. 
Thanks.